packages/protocol/test/backward/version.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  ContractVersion,
+  ContractVersionDelta,
+  Delta,
+  DeltaUtil,
+} from '@celo/protocol/lib/backward/version'
+import { assert } from 'chai'
+
+describe('#version()', () => {
```

Comment:
Is there a `version()` function we're testing? Maybe just have top-level `describe(<class>)` for the different classes you're testing from this library.